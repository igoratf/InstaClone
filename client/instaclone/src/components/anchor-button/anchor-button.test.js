import React from 'react';
import { shallow } from 'enzyme';
import AnchorButton from './anchor-button.component';

describe('AnchorButton component', () => {
  it('should render AnchorButton', () => {
    expect(shallow(<AnchorButton />)).toMatchSnapshot();
  })
})