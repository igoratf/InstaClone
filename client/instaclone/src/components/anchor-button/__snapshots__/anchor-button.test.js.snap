// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnchorButton component should render AnchorButton 1`] = `ShallowWrapper {}`;
